plugins/modules/powervm_dlpar.py

```diff
@@ -19,8 +19,9 @@
     - Navinakumar Kandakur(@nkandak1)
 short_description: Dynamically managing resources of partition
 notes:
-    - In case of update_pv action if updation of settings belongs to atleast one VIOS is successful then
-      changed status shown as 1 and failed updates are shown as warning, This behavior is same for update_npiv and update_vod also.
+    - In case of update_pv action,  If updating of settings belongs to at least one VIOS is successful then
```

Comment:
If the updating of settings for at least one VIOS is successful during the execution of the "update_pv" action, the changed status will be displayed as 1, and any failed updates will be shown as warnings. This behavior remains consistent for both the "update_npiv" and "update_vod" actions as well.

Reply:
Fixed